test-requirements.txt

```diff
@@ -5,4 +5,4 @@ paramiko
 tornado<5
 salt
 pywinrm
-ansible>=2.4,<3
```

Comment:
As I think mentioned elsewhere, this change needs to leave ansible in here, otherwise it's not installed.  I'm seeing this trying to run this change with tox